Console spam when right clicking hostile mobs

### Observed Behavior

Console spam when right clicking any hostile mobs (Creeper, Zombie, Ghast, Spider,...)

### Expected Behavior

No spamming.

### Reproduction steps

1. Use your hand and right click in any hostile mobs
2. Observe console spam in...console.

### Stack trace or error log

```shell
[11:04:46 ERROR]: Could not pass event PlayerInteractEntityEvent to GriefPrevention v16.18.6
java.lang.NoClassDefFoundError: org/bukkit/entity/CopperGolem
        at GriefPrevention.jar//me.ryanhamshire.GriefPrevention.PlayerEventHandler.onPlayerInteractEntity(PlayerEventHandler.java:1330) ~[?:?]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:71) ~[leaf-api-1.21.8-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:59) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:629) ~[leaf-api-1.21.8-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl$1.performInteraction(ServerGamePacketListenerImpl.java:2905) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.network.ServerGamePacketListenerImpl$1.onInteraction(ServerGamePacketListenerImpl.java:2955) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket$InteractionAction.dispatch(ServerboundInteractPacket.java:140) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.dispatch(ServerboundInteractPacket.java:91) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleInteract(ServerGamePacketListenerImpl.java:2895) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:78) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:14) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:27) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:155) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1484) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:165) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:129) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1465) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1459) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:139) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1414) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1423) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1303) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:298) ~[leaf-1.21.8.jar:1.21.8-175-e30491f]
        at java.base/java.lang.Thread.run(Thread.java:1474) ~[?:?]
Caused by: java.lang.ClassNotFoundException: org.bukkit.entity.CopperGolem
        ... 27 more
```

### Server version

```shell
> version
[11:06:47 INFO]: Checking version, please wait...
[11:06:49 INFO]: This server is running Leaf version 1.21.8-175-ver/1.21.8@e30491f (2026-02-08T19:16:20Z) (Implementing API version 1.21.8-R0.1-SNAPSHOT)
* You are running the latest version
Previous: 1.21.8-174-25ec665 (MC: 1.21.8)
```

### GriefPrevention version

```shell
> version GriefPrevention
[11:07:10 INFO]: GriefPrevention version 16.18.6
```

### Configuration

```yaml
# Default values are perfect for most servers.  If you want to customize and have a question, look for the answer here first: http://dev.bukkit.org/bukkit-plugins/grief-prevention/pages/setup-and-configuration/

GriefPrevention:
  SeaLevelOverrides:
    world_iris: -1
    world_iris_nether: -1
    world_iris_the_end: -1
    world: -1
    donateworld: -1
    world_lobby: -1
    world_pvp: -1
    dimensionalhome: -1
  Claims:
    Mode:
      world_iris_the_end: Disabled
      world_pvp: Survival
      donateworld: Survival
      dimensionalhome: Survival
      world_lobby: Survival
      world_iris: Survival
      world: Survival
      world_iris_nether: Disabled
    PreventGlobalMonsterEggs: true
    PreventTheft: true
    HoppersRequireBuildTrust: false
    ProtectCreatures: true
    PreventButtonsSwitches: true
    LockWoodenDoors: true
    LockTrapDoors: true
    LockFenceGates: true
    EnderPearlsRequireAccessTrust: true
    RefundDeniedEnderPearls: true
    RaidTriggersRequireBuildTrust: true
    ProtectHorses: true
    ProtectDonkeys: true
    ProtectLlamas: true
    InitialBlocks: 5000
    Claim Blocks Accrued Per Hour:
      Default: 999
    Max Accrued Claim Blocks:
      Default: 100000
    Accrued Idle Threshold: 0
    AccruedIdlePercent: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: 4
    AutomaticNewPlayerClaimsRadiusMinimum: 0
    ExtendIntoGroundDistance: 5
    MinimumWidth: 5
    MinimumArea: 100
    MaximumDepth: -2147483648
    InvestigationTool: STICK
    ModificationTool: GOLDEN_SHOVEL
    Expiration:
      ChestClaimDays: 7
      UnusedClaimDays: 14
      AllClaims:
        DaysInactive: 60
        ExceptWhenOwnerHasTotalClaimBlocks: 10000
        ExceptWhenOwnerHasBonusClaimBlocks: 5000
      AutomaticNatureRestoration:
        SurvivalWorlds: false
    AllowTrappedInAdminClaims: false
    MaximumNumberOfClaimsPerPlayer: 0
    CreationRequiresWorldGuardBuildPermission: true
    VillagerTradingRequiresPermission: true
    CommandsRequiringAccessTrust: /sethome
    DeliverManuals: true
    ManualDeliveryDelaySeconds: 30
    RavagersBreakBlocks: true
    FireSpreadsInClaims: false
    FireDamagesInClaims: false
    LecternReadingRequiresAccessTrust: true
  Spam:
    Enabled: false
    LoginCooldownSeconds: 10
    LoginLogoutNotificationsPerMinute: 5
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: /tell;/pm;/r;/whisper;/msg
    WarningMessage: Bạn ồn ào quá rồi đấy ! Im lặng đi !
    BanOffenders: false
    BanMessage: Banned vì spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 120
    Logout Message Delay In Seconds: 0
  PvP:
    RulesEnabledInWorld:
      world_iris: true
      world_iris_nether: true
      world_iris_the_end: true
      world: true
      donateworld: true
      world_lobby: true
      world_pvp: true
      dimensionalhome: true
    ProtectFreshSpawns: true
    PunishLogout: false
    CombatTimeoutSeconds: 8
    AllowCombatItemDrop: false
    BlockedSlashCommands: /home;/spawn;/tpa;/hub;/rtp;/tpa;/gptp
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: true
      AdministrativeClaims: true
      AdministrativeSubdivisions: true
    AllowLavaDumpingNearOtherPlayers:
      PvPWorlds: true
      NonPvPWorlds: false
    AllowFlintAndSteelNearOtherPlayers:
      PvPWorlds: true
      NonPvPWorlds: false
    ProtectPetsOutsideLandClaims: false
  Economy:
    ClaimBlocksMaxBonus: 0
    ClaimBlocksPurchaseCost: 0.0
    ClaimBlocksSellValue: 0.0
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: true
  BlockLandClaimExplosions: true
  BlockSurfaceCreeperExplosions: true
  BlockSurfaceOtherExplosions: true
  LimitSkyTrees: true
  LimitTreeGrowth: false
  PistonMovement: CLAIMS_ONLY
  PistonExplosionSound: true
  FireSpreads: false
  FireDestroys: false
  AdminsGetWhispers: true
  AdminsGetSignNotifications: true
  VisualizationAntiCheatCompatMode: false
  SmartBan: true
  Mute New Players Using Banned Words: false
  MaxPlayersPerIpAddress: 2
  SilenceBans: true
  Siege:
    Worlds: []
    BreakableBlocks:
    - GRASS_BLOCK
    - DIRT
    - COBBLESTONE
    - OAK_PLANKS
    - SPRUCE_PLANKS
    - BIRCH_PLANKS
    - JUNGLE_PLANKS
    - ACACIA_PLANKS
    - DARK_OAK_PLANKS
    - SAND
    - GRAVEL
    - GLASS
    - GRASS     <-- can't understand this entry, see BukkitDev documentation
    - FERN
    - DEAD_BUSH
    - WHITE_WOOL
    - ORANGE_WOOL
    - MAGENTA_WOOL
    - LIGHT_BLUE_WOOL
    - YELLOW_WOOL
    - LIME_WOOL
    - PINK_WOOL
    - GRAY_WOOL
    - LIGHT_GRAY_WOOL
    - CYAN_WOOL
    - PURPLE_WOOL
    - BLUE_WOOL
    - BROWN_WOOL
    - GREEN_WOOL
    - RED_WOOL
    - BLACK_WOOL
    - SNOW
    - GLASS_PANE
    DoorsOpenDelayInSeconds: 300
    CooldownEndInMinutes: 60
  EndermenMoveBlocks: false
  SilverfishBreakBlocks: false
  CreaturesTrampleCrops: false
  RabbitsEatCrops: true
  HardModeZombiesBreakDoors: false
  MobProjectilesChangeBlocks: false
  UseBanCommand: false
  BanCommandPattern: ban %name% %reason%
  Advanced:
    fixNegativeClaimblockAmounts: true
    ClaimExpirationCheckRate: 60
    OfflinePlayer_cache_days: 90
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false
  ConfigVersion: 1
```

### Plugin list

```shell
> pl
[11:14:38 INFO]: ℹ Server Plugins (5):
[11:14:38 INFO]: Paper Plugins (0):
[11:14:38 INFO]: Bukkit Plugins (5):
[11:14:39 INFO]:  - GriefPrevention, PlaceholderAPI, ProtocolLib, ViaBackwards, ViaVersion
```

### Running without GriefPrevention

- [x] I attempted running the server without GriefPrevention installed.
- [x] The problem does _not_ occur when GriefPrevention is removed from the server.

### Running with only GriefPrevention

- [x] I attempted running only GriefPrevention on the server.
- [x] The issue still occurs when GriefPrevention is the only plugin running.

### Running on a fresh, clean server installation

- [x] I attempted testing for the issue on a new server.
- [x] The issue still occurs on a new server.

### Using unmodified client

- [x] I attempted testing for the issue with the vanilla client.
- [x] The issue still occurs when using the vanilla client.

### We appreciate you taking the time to fill out a bug report!

- [x] I searched for similar issues before submitting this bug report.

--- a/src/main/resources/plugin.yml
+++ b/src/main/resources/plugin.yml
@@ -3,7 +3,7 @@ main: me.ryanhamshire.GriefPrevention.GriefPrevention
 softdepend: [Vault, Multiverse-Core, My_Worlds, MystCraft, Transporter, WorldGuard, WorldEdit, RoyalCommands, MultiWorld, Denizen, CommandHelper, Iris]
 dev-url: https://dev.bukkit.org/projects/grief-prevention
 version: '${git.commit.id.describe}'
-api-version: '1.21'
+api-version: '1.21.10'
 commands:
     abandonclaim:
       description: Deletes a claim.

